### PR TITLE
refactor: Enable magic constant linter

### PR DESCRIPTION
**Pull request by victorcolombo**
## Is this change user facing?
NO

## References (if applicable):

Part of https://github.com/kurtosis-tech/kurtosis/issues/1007

### CODE REVIEW

File: cli/cli/commands/enclave/inspect/inspect.go [hunk 1]
```diff
@@ -166,8 +166,8 @@ func PrintEnclaveInspect(ctx context.Context, kurtosisBackend backend_interface.
 			return stacktrace.NewError("No printing function found for enclave object '%v'; this is a bug in Kurtosis!", header)
 		}
 
-		numRunesInHeader := utf8.RuneCountInString(header) + 2 // 2 because there will be a space before and after the header
-		numPadChars := (headerWidthChars - numRunesInHeader) / 2
+		numRunesInHeader := utf8.RuneLen(' ') + utf8.RuneCountInString(header) + utf8.RuneLen(' ') // there will be a space before and after the header
+		numPadChars := (headerWidthChars - numRunesInHeader) / 2                                   //nolint:gomnd
```
Comment (Peeeekay) [inline cli/cli/commands/enclave/inspect/inspect.go:170]: why are we not using lint here?

File: container-engine-lib/lib/network_helpers/network_helpers.go [hunk 2]
```diff
@@ -27,8 +29,8 @@ func GetFreeIpAddrFromSubnet(takenIps map[string]bool, subnet *net.IPNet) (net.I
 	// The IP can be either 4 bytes or 16 bytes long; we need to handle both!
 	// See https://gist.github.com/ammario/649d4c0da650162efd404af23e25b86b
 	var intIp uint32
-	if len(subnetIp) == 16 {
-		intIp = binary.BigEndian.Uint32(subnetIp[12:16])
+	if len(subnetIp) == net.IPv6len {
+		intIp = binary.BigEndian.Uint32(subnetIp[12:net.IPv6len])
```
Comment (Peeeekay) [inline container-engine-lib/lib/network_helpers/network_helpers.go]: we can use constant for `12` as well.

File: core/server/api_container/server/startosis_engine/builtins/kurtosis_module.go [hunk 3]
```diff
@@ -15,6 +15,11 @@ const (
 	connectionSubmoduleAllowedAttrName = "ALLOWED"
 )
 
+var (
+	totalPacketLossPercentage = starlark.Float(100) //nolint:gomnd
+	noPacketLossPercentage    = starlark.Float(0)
```
Comment (Peeeekay) [inline core/server/api_container/server/startosis_engine/builtins/kurtosis_module.go]: possibly nit:

packageLostPercentage = 100

and in the code we could just do starlark.Float(packageLostPercentage)

also why don't we need `//nolint` on line `20`

**Review by Peeeekay — APPROVED** [review]

File: container-engine-lib/lib/network_helpers/network_helpers.go (hunk 2 above)
Comment (victorcolombo) [reply container-engine-lib/lib/network_helpers/network_helpers.go]: Done!

File: cli/cli/commands/enclave/inspect/inspect.go (hunk 1 above)
Comment (victorcolombo) [reply cli/cli/commands/enclave/inspect/inspect.go:170]: no good way to define this two. It's clear that we are averaging and defining `const consantToTakeAverage = 2` seems overkill :)

File: core/server/api_container/server/startosis_engine/builtins/kurtosis_module.go (hunk 3 above)
Comment (victorcolombo) [reply core/server/api_container/server/startosis_engine/builtins/kurtosis_module.go]: Done!

**Pull request merged**